client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -13,66 +14,97 @@ import {
 	PaymentsDataChargeTooltip,
 	PaymentsDataFeesTooltip,
 } from './payments-data-highlights-tooltips';
+import { usePaymentActivityData } from 'wcpay/data';
+import { getAdminUrl } from 'wcpay/utils';
+import { DateRange, PaymentsActivityData } from './interfaces';
 
 import './style.scss';
-import { getAdminUrl } from 'wcpay/utils';
+
+/**
+ * This will be replaces in the future with a dynamic date range picker.
+ */
+const getDateRange = (): DateRange => {
+	return {
+		// Subtract 7 days from the current date.
+		date_start: moment()
+			.subtract( 7, 'd' )
+			.format( 'YYYY-MM-DD\\THH:mm:ss' ),
+		date_end: moment().format( 'YYYY-MM-DD\\THH:mm:ss' ),
+	};
+};
 
 const PaymentsActivityData: React.FC = () => {
+	const {
+		paymentActivityData: {
+			total_payments_volume: totalPaymentsVolume,
+			charges,
+			fees,
+			disputes,
+			refunds,
+		},
+		isLoading,
+	} = usePaymentActivityData( getDateRange() ) as PaymentsActivityData;
```

Comment:
⛏️ Nit: avoid type assertions (`as`) and prefer to use the inferred type from `usePaymentActivityData`.

Reply:
Can you please give me an example of what you mean?

Reply:
Apologies, I should've added a suggestion.

```suggestion
	} = usePaymentActivityData( getDateRange() );
```

This will mean we have to handle the case that the hook returns `undefined`, but that's a good thing – handle what could be a runtime error.

Reply:
Here's a little bit about this in the TS guidelines (recently merged): [TypeScript - General Conventions - Avoid type assertions](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/general-conventions.md#avoid-type-assertions).

Reply:
Hmm, makes sense fixed it!